haystack/nodes/file_converter/pdf.py

```diff
@@ -169,15 +178,28 @@ def _read_pdf(self, file_path: Path, layout: bool, encoding: Optional[str] = Non
                        the content stream order.
         :param encoding: Encoding that overwrites self.encoding and will be passed as `-enc` parameter to `pdftotext`.
                          (See list of available encodings by running `pdftotext -listenc` in the terminal)
+        :param start_page: The page number where to start the conversion
+        :param end_page: The page number where to end the conversion.
         """
         if not encoding:
             encoding = self.encoding
 
         command = ["pdftotext", "-enc", str(encoding), "-layout" if layout else "-raw", str(file_path), "-"]
+
+        start_page = start_page or 1
+        command.insert(3, "-f")
+        command.insert(4, str(start_page))
```

Comment:
Could we add this directly into the assembly of the `command` above? So something like 
```python
command = ["pdftotext", "-enc", str(encoding), "-f", start_page if start_page else 1,"-layout" if layout else "-raw", str(file_path), "-"]
```
I think avoiding using `inserts` would be good so future developers won't have to realize that they made need to update the insertion indices if they ever change the `command` list.

Reply:
Ok, I did just to be more readable. Changing it.